@xen-orchestra/backups/writers/_AbstractBackupWriter.js

```diff
@@ -0,0 +1,25 @@
+const { createLogger } = require('@xen-orchestra/log')
+
+const { getVmBackupDir } = require('../_getVmBackupDir')
+
+const { debug } = createLogger('xo:backups:AbstractBackupWriter')
+
+exports.AbstractBackupWriter = (BaseClass = Object) =>
```

Comment:
I think we should rename it to `MixinBackupWriter` to make things clearer, what's your opinion on that?

Reply:
I agree :+1: